src/parser/smt2/smt2.cpp

```diff
@@ -1160,11 +1178,10 @@ Type Smt2::processSygusNestedGTerm( int sub_dt_index, std::string& sub_dname, st
           d_sygus_bound_var_type[sop] = t;
         }
       }
-    }else{
+    }
+    else
+    {
       std::vector< Expr > children;
-      if( sop.getKind() != kind::BUILTIN ){
-        children.push_back( sop );
-      }
```

Comment:
Why is it OK to remove this?

Reply:
Please ignore, I thought I had deleted this

Reply:
It was deleted, but is now covered by `applyParseOp` below.

Reply:
No, I mean I thought I had deleted the comment. :)

Reply:
Hahah ok :)